coursefunctions.h

```diff
@@ -1,4 +1,5 @@
 #include "definitions.h"
+#include "utilitiesfunctions.h"
```

Comment:
Super cool that you were able to extract out the Game of Life code that wasn't important for the pipeline into a separate file. I suggest not even including it since it will be unnecessary for the rest of the semester